<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>625</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>口罩人脸识别系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="MAreaName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>监控区域</string>
    </property>
   </widget>
   <widget class="QLabel" name="IAreaName">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>证件照</string>
    </property>
   </widget>
   <widget class="QLabel" name="recResult">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>380</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="date">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border-width: 0;
border-style: outset;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;日期时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="MonitorArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="IdArea">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>150</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="subMonitorArea">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>150</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="IAreaName_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>监控照</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>540</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>证件照</string>
    </property>
    <addaction name="openID"/>
    <addaction name="resetID"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>监控照</string>
    </property>
    <addaction name="openCamera"/>
    <addaction name="closeCamera"/>
    <addaction name="separator"/>
    <addaction name="takePhoto"/>
    <addaction name="savePhoto"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>人脸识别</string>
    </property>
    <addaction name="faceRecog"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="helpBook"/>
    <addaction name="aboutMe"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openID">
   <property name="text">
    <string>上传(O)</string>
   </property>
  </action>
  <action name="resetID">
   <property name="text">
    <string>重置(R)</string>
   </property>
  </action>
  <action name="openCamera">
   <property name="text">
    <string>打开摄像(O)</string>
   </property>
  </action>
  <action name="closeCamera">
   <property name="text">
    <string>关闭摄像(C)</string>
   </property>
  </action>
  <action name="takePhoto">
   <property name="text">
    <string>抓拍(P)</string>
   </property>
  </action>
  <action name="savePhoto">
   <property name="text">
    <string>保存(S)</string>
   </property>
  </action>
  <action name="faceRecog">
   <property name="text">
    <string>验证(V)</string>
   </property>
  </action>
  <action name="action_I">
   <property name="text">
    <string>证件照(I)</string>
   </property>
  </action>
  <action name="action_O_3">
   <property name="text">
    <string>监控照(O)</string>
   </property>
  </action>
  <action name="helpBook">
   <property name="text">
    <string>使用说明(H)</string>
   </property>
  </action>
  <action name="aboutMe">
   <property name="text">
    <string>关于(R)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
